web/app/themes/ppj/src/img/svg/facebook.svg

```diff
@@ -1,9 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Generator: Adobe Illustrator 22.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 246.9 246.9" style="enable-background:new 0 0 246.9 246.9;" xml:space="preserve">
-
-<path class="st0" d="M233.3,0H13.6C6.1,0,0,6.1,0,13.6v219.7c0,7.5,6.1,13.6,13.6,13.6h118.3v-95.6H99.7V114h32.2V86.6
-	c0-31.9,19.5-49.3,47.9-49.3c13.6,0,25.3,1,28.8,1.5v33.3l-19.7,0c-15.5,0-18.5,7.4-18.5,18.1V114h36.9l-4.8,37.3h-32.1v95.6h62.9
-	c7.5,0,13.6-6.1,13.6-13.6V13.6C246.9,6.1,240.8,0,233.3,0z"/>
+<svg
+  aria-labelledby="facebookIconTitle facebookIconDesc"
+  style="enable-background:new 0 0 246.9 246.9;"
+  viewBox="0 0 246.9 246.9"
+  x="0px"
+  xml:space="preserve"
+  xmlns="http://www.w3.org/2000/svg"
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  y="0px"
+>
+  <title id="facebookIconTitle">Facebook icon</title>
+  <desc id="facebookIconDesc">A navy blue F character on a white background</desc>
```

Comment:
A better description might be "An icon representing the Facebook logo" – rather than describing the logo itself 🤷🏻‍♂️

Reply:
Will discuss with Josh.